projects/bob-gateway/index.js

```diff
@@ -31,10 +31,7 @@ module.exports = {
       return sumTokens2({
         api,
         chain: "bob",
-        tokens: [
-          ADDRESSES.bob.WBTC,
-          ADDRESSES.bob.TBTC,
-        ],
+        tokens: Object.values(ADDRESSES.bob),
```

Comment:
hmm, this leads unexpected behavior if some key is removed from coreAssets file

Reply:
> hmm, this leads unexpected behavior if some key is removed from coreAssets file

Good point. I'll revert that change.